lib/db/set.go

```diff
@@ -531,5 +531,6 @@ func fatalError(err error, opStr string, db *Lowlevel) {
 			}
 		}
 	}
-	warnAndPanic(fmt.Errorf("%v: %w:", opStr, err))
+	l.Warnf("Fatal error: %v: %v", opStr, err)
```

Comment:
Oh, it's in the `opStr` here is it? If so, all good.

Reply:
Exactly, it contains info on method and the problematic arguments, and baking it into the error here made it turn up in the panic message.